Makefile

```diff
@@ -68,8 +68,8 @@ bin:
 dep:
 	export DEBIAN_FRONTEND=noninteractive && \
 	export TZ=America/New_York && \
-	apt-get update && \
 	if [ "$(TARGET)" = "jetson" ]; then \
+		apt-get update && \
```

Comment:
why move update in jetson only?

Reply:
Moving apt-get update from before the conditional.

- on jetson, just apt update immediately
- on pi, swap backports, then apt update